Library/Social Engineering/Social Engineering Modifiers.adm

```diff
@@ -0,0 +1,17 @@
+{
+	"type": "modifier_list",
+	"version": 2,
+	"id": "981785b6-2410-47bb-ad46-932ac21e6a50",
+	"rows": [
+		{
+			"type": "modifier",
+			"id": "39ce3558-7397-40b5-8cf8-1d51b7efb9da",
+			"name": "Aspected",
+			"reference": "SE77",
+			"cost_type": "percentage",
+			"cost": -20,
+			"affects": "total",
+			"notes": "Choose aspect"
```

Comment:
Typically this is done using replaceable text, i.e. something like "@Aspect@", which will then prompt the user to enter in a value when they try to apply it to a character sheet. You can put these in the name or notes field.

Reply:
@Coppini were you planning to update this?

Reply:
```suggestion
			"notes": "@Aspect@"
```
Sorry, hadn't noticed your comment before.

Reply:
Updated 👍🏻